contracts/examples/Mars.sol

```diff
@@ -294,18 +308,32 @@ contract RevertingBytesMars is Mars {
 contract RevertingEmptyMars is Mars {
     constructor(IbcDispatcher _dispatcher) Mars(_dispatcher) {}
 
-    function onRecvPacket(IbcPacket memory) external view override onlyIbcDispatcher returns (AckPacket memory ack) {
+    function onRecvPacket(IbcPacket memory)
+        external
+        view
+        override
+        onlyIbcDispatcher
+        returns (AckPacket memory ack, bool skipAck)
+    {
         // solhint-disable-next-line
         require(false);
         ack = AckPacket(false, "");
+        skipAck = false;
     }
 }
 
 contract PanickingMars is Mars {
     constructor(IbcDispatcher _dispatcher) Mars(_dispatcher) {}
 
-    function onRecvPacket(IbcPacket memory) external view override onlyIbcDispatcher returns (AckPacket memory ack) {
+    function onRecvPacket(IbcPacket memory)
+        external
+        view
+        override
+        onlyIbcDispatcher
+        returns (AckPacket memory ack, bool skipAck)
```

Comment:
would it be more intuitive for users if instead of returning whether or not to skip, the dapp returns wether or not to process the next ack?